python/ray/serve/tests/unit/test_deployment_state.py

```diff
@@ -5,6 +5,7 @@
 
```

Comment:
I might have missed it, but could you make sure to add tests that check whether deleted deployments' autoscaling information is actually removed?